.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "plugins/SRAW"]
 	path = plugins/SRAW
 	url = git@hpcgitlab.hpc.inl.gov:RAVEN_PLUGINS/SRAW.git
+[submodule "plugins/FARM"]
+	path = plugins/FARM
+	url = git@github.com:Argonne-National-Laboratory/FARM.git
```

Comment:
This isn't a fix for this PR, but using ssh instead of https has been causing issues externally since we switched to it, though it does make internal to INL (e.g. testing machines) work more smoothly.